setup.py

```diff
@@ -33,10 +33,12 @@
     'scikit-learn>=0.19.1',
     'flask>=0.12.2',
     'GPyOpt>=1.2.5',
-    'enum34',
-    'matplotlib',
+    'matplotlib'
 ]
 
+if sys.version_info > (3, 4):
```

Comment:
Hi @LarsHH , shouldn't this logic be reversed?

If the Python version is _less than_ 3.4, only then do we need to include the `enum34` package in order to backport the enum features introduced in 3.4 into the older language. 

For Python >= 3.4, we don't want to install the `enum34`, since it is redundant and doing so [will introduce other problems](https://stackoverflow.com/questions/43124775/why-python-3-6-1-throws-attributeerror-module-enum-has-no-attribute-intflag).

```python
if sys.version_info < (3, 4):
    REQUIRED += ['enum34']
```